src/databricks/labs/ucx/install.py

```diff
@@ -249,6 +249,12 @@ def _configure_new_installation(self) -> WorkspaceConfig:
 
         # Check if terraform is being used
         is_terraform_used = self._prompts.confirm("Do you use Terraform to deploy your infrastructure?")
+
+        # Flag to check if the assessment workflow has to be run after installation
+        run_assessment_workflow = self._prompts.confirm(
```

Comment:
do this prompt later in the code - after we display the readme link

Reply:
@nfx  readme link is getting displayed in WorkspaceInstallation after all the job is getting created. If I put after the readme every time it will ask for prompt every time.